### PR TITLE
add deploy preview to CONTRIBUTING.md 

### PR DESCRIPTION
**Pull request by barriebyron**
Today I learned that every ready-for-review PR has a commit status for a netlify deployed preview. Click Details and !voila! instant preview for that PR. I updated our CONTRIBUTING.md file so that others can also learn about this awesome sauce.

### CODE REVIEW

**Comment by toschdev** [comment]
Thanks, Barrie for adding this to the CONTRIBUTING file  🙏

File: tutorials.code-workspace [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"folders": [
 		{
-			"path": "."
+			"path": "connecting-to-testnet"
```
Comment (toschdev) [inline tutorials.code-workspace:4]: This is for your local workspace? Is this helpful for others to have this file `tutorials.code-worspace` in the repository or should we consider adding this file to the `.gitignore`?

Comment (barriebyron) [reply tutorials.code-workspace:4]: I like the .gitignore since workspaces are personal. wdyt?

Comment (toschdev) [reply tutorials.code-workspace:4]: Yes, let's add it in this PR

Comment (barriebyron) [inline tutorials.code-workspace:4]: @toschdev hmmm how to make this change not part of the PR?

[the remark this answers is not in the document]
Comment (toschdev) [reply tutorials.code-workspace:4]: I'd prefer if the file could be deleted in this PR

**Review by toschdev — APPROVED** [review]

File: tutorials.code-workspace (hunk 1 above)
[the remark this answers is not in the document]
Comment (barriebyron) [reply tutorials.code-workspace]: oof I meant to do that, thank you!
